netcode.c

```diff
@@ -4172,6 +4210,10 @@ int skillz_add_client_to_match(struct netcode_server_t * server, uint64_t skillz
         match->num_clients_in_match++;
         server->skillz_match_id[client_index] = skillz_match_id;
 
+        if( match->start_time == 0 )
+            match->start_time = server->time;
+        match->last_restart_time = server->time;
```

Comment:
Is this really a match restart time?  Seems more like `last_client_connect_time`...

Reply:
I can rename it.  I was just keeping it short but you are right it is misleading.